### PR TITLE
made a card component for project showcase and many design changes

### PR DESCRIPTION
**Pull request by revengemiroz**

### CODE REVIEW

File: client/src/pages/Activated/Activated.js [hunk 1]
```diff
@@ -167,15 +137,15 @@ function Activated(props) {
...
+                                <a target='_blank' rel="noopener noreferrer">Live Link</a>
+                                <a target='_blank' rel="noopener noreferrer">Repo Link</a>
+                                <a target='_blank' rel="noopener noreferrer">Contact</a>
                             </Links>
 
                             <div className='imgContainer'>
                                 <Zoom wrapStyle={{ display: 'block' }}>
 
-                                    <img src={IMG_Social} width='100%' height='100%'></img>
+                                    <img alt='aasdasd' src={IMG_Social} width='100%' height='100%'></img>
```
Comment (ivanms1) [inline client/src/pages/Activated/Activated.js]: ? something more meaningful please lol

File: client/src/pages/Activated/Activated.js [hunk 2]
```diff
@@ -193,15 +163,15 @@ function Activated(props) {
...
```
Comment (ivanms1) [inline client/src/pages/Activated/Activated.js]: same

File: client/src/pages/Home/Home.js [hunk 3]
```diff
@@ -50,13 +63,9 @@ function Home() {
 
 
   function deleteuserProject(projectId) {
-    if (window.confirm('Are you sure you want to delete it ?')) {
```
Comment (ivanms1) [inline client/src/pages/Home/Home.js:53]: I suggest using a modal, you could create a component that you can reuse everywhere

File: client/src/pages/Home/style.js [hunk 4]
```diff
@@ -128,47 +142,56 @@ background-color:${props => props.active ? '#00CB5B' : '#ED2C49'};
...
+      flex-direction: column;
+      justify-content: space-between;
+      align-items: center;
+      width:100%;
+      height:100%;
+      max-width: 526px;
+      max-height: 208px;
+      border-radius: 10px;
+      border: none;
+      box-Shadow: 10px 10px 40px 4px rgba(45,78,255,0.15);
```
Comment (ivanms1) [inline client/src/pages/Home/style.js]: does this still work capitalized? anyways please change it

File: client/src/pages/SubmitYourProject/style.js [hunk 5]
```diff
@@ -15,23 +21,32 @@ background-color:#F7F8FC;
...
+        color:#000000;
     }
     span:last-child{
-        color:#E85F99;
+        color:#000000;
     }
 }
+
+@media screen and (max-width:600px){
+   >p{
```
Comment (ivanms1) [inline client/src/pages/SubmitYourProject/style.js]: > p? what does this apply to?

File: client/src/pages/Home/style.js (hunk 4 above)
Comment (revengemiroz) [reply client/src/pages/Home/style.js]: resolved

File: client/src/pages/Home/Home.js (hunk 3 above)
Comment (revengemiroz) [reply client/src/pages/Home/Home.js:53]: resolved. i think i pushed before making this change

File: client/src/pages/Activated/Activated.js (hunk 2 above)
Comment (revengemiroz) [reply client/src/pages/Activated/Activated.js]: resolved

File: client/src/pages/Activated/Activated.js (hunk 1 above)
Comment (revengemiroz) [reply client/src/pages/Activated/Activated.js]: resolved

File: client/src/pages/SubmitYourProject/style.js (hunk 5 above)
Comment (revengemiroz) [reply client/src/pages/SubmitYourProject/style.js]: the header
image
"showcase them ....."

**Review by ivanms1 — APPROVED** [review]
good job

**Pull request merged**